frontend/src/pages/Organizations/Organizations.tsx

```diff
@@ -1,267 +1,105 @@
-        body
-      });
-      setOrganizations(organizations.concat(org));
+      let rows = await apiGet<Organization[]>('/organizations/');
+      setOrganizations(rows);
     } catch (e) {
-      setErrors({
-        global:
-          e.status === 422
-            ? 'Error when submitting organization entry.'
-            : e.message ?? e.toString()
-      });
       console.error(e);
     }
-  };
+  }, [apiGet]);
 
   React.useEffect(() => {
-    document.addEventListener('keyup', (e) => {
-      //Escape
-      if (e.keyCode === 27) {
-        setShowModal(false);
-      }
-    });
-  }, [apiGet]);
+    fetchOrganizations();
+  }, [apiGet, fetchOrganizations]);
 
   return (
-    <div className={classes.root}>
-      <h1>Organizations</h1>
-      <Table<Organization>
-        columns={columns}
-        data={organizations}
-        fetchData={fetchOrganizations}
-      />
-      {user?.userType === 'globalAdmin' && (
-        <>
-          <h2>Add an organization</h2>
-          {errors.global && <p className={classes.error}>{errors.global}</p>}
-          <OrganizationForm
-            onSubmit={onSubmit}
-            type="create"
-          ></OrganizationForm>
-          <ImportExport<Organization>
-            name="organizations"
-            fieldsToExport={[
-              'name',
-              'rootDomains',
-              'ipBlocks',
-              'isPassive',
-              'inviteOnly'
-            ]}
-            onImport={async (results) => {
-              // TODO: use a batch call here instead.
-              let createdOrganizations = [];
-              for (let result of results) {
-                createdOrganizations.push(
-                  await apiPost('/organizations/', {
-                    body: {
-                      ...result,
-                      // These fields are initially parsed as strings, so they need
-                      // to be converted to arrays.
-                      ipBlocks: (
-                        ((result.ipBlocks as unknown) as string) || ''
-                      ).split(','),
-                      rootDomains: (
-                        ((result.rootDomains as unknown) as string) || ''
-                      ).split(',')
-                    }
-                  })
-                );
-              }
-              setOrganizations(organizations.concat(...createdOrganizations));
-            }}
-            getDataToExport={() => organizations}
-          />
-        </>
-      )}
-      {showModal && (
-        <div>
-          <Overlay />
-          <ModalContainer>
-            <Modal
-              actions={
-                <>
-                  <Button
-                    outline
-                    type="button"
-                    onClick={() => {
-                      setShowModal(false);
-                    }}
-                  >
-                    Cancel
-                  </Button>
-                  <Button
```

Comment:
Why remove the delete organization functionality?

If we want to restrict this to, maybe, global admins, or remove it altogether, we should also probably restrict / remove the DELETE API endpoint accordingly.

Reply:
This was just because there wasn't functionality in the mockup to delete organizations. I added this as a TODO to allow deleting organizations, which I'll add in to the next PR adding teams.